src/Docfx.MarkdigEngine.Extensions/MarkdownExtensions.cs

```diff
@@ -56,9 +57,34 @@ public static class MarkdownExtensions
             return pipeline;
         }
 
+        ReplaceExistingExtensions();
+
         pipeline.Configure(string.Join("+", optionalExtensions));
 
         return pipeline;
+
+        // Some markdig extensions are added by UseDocfxExtensions with non-default settings.
+        // This methods replace these extensions with default settings.
+        void ReplaceExistingExtensions()
```

Comment:
These feel like configuration of existing extensions, it might be more future proof if we configure it such as:

```
markdigExtensions: [
    'other-extensions'
    { emoji: { enableSmileys: true }],
    { autoidentifier: 'default' }
]
```

Reply:
I've added commented features to override extension properties by PR #9820.

**Example configs**

```
markdigExtensions: [
    "FootNotes",
    { "Emojis": { "options": {  "enableSmileys": true } },
    { "AutoIdentifiers": { "options": 'default '} },
    { "MediaLinks": { "options": { "width": 800, "height": 400 }} 
]
```

Note: I've added `options` key. Because it might required to support additional property
when implementing **`remove` markdig extensions that are enabled by default** feature.(Discussed at #7833)